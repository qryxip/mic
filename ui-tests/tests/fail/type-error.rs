use mic::answer;

#[answer]
fn main() -> _ {
    vec![1, 2, 3]
}
