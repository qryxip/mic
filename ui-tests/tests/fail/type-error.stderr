error[E0277]: `Vec<{integer}>` doesn't implement `std::fmt::Display`
 --> tests/fail/type-error.rs:3:1
  |
3 | #[answer]
  | ^^^^^^^^^ `Vec<{integer}>` cannot be formatted with the default formatter
  |
  = help: the trait `std::fmt::Display` is not implemented for `Vec<{integer}>`
  = note: in format strings you may be able to use `{:?}` (or {:#?} for pretty-print) instead
  = note: this error originates in the macro `$crate::format_args_nl` (in Nightly builds, run with -Z macro-backtrace for more info)
